cmd/manager/scap.go

```diff
@@ -300,6 +300,8 @@ func fetch(client *kubernetes.Clientset, objects []utils.ResourcePath) (map[stri
 				DBG("Encountered non-fatal error to be persisted in the scan: %s", err)
 				objerr := fmt.Errorf("could not fetch %s: %w", uri, err)
 				warnings = append(warnings, objerr.Error())
+				// additionally, we'll add a warning comment in the object so openSCAP can read and process it
+				results[rpath.DumpPath] = []byte("# Error=" + kerrors.ReasonForError(err))
```

Comment:
Looks good! Let's make this look a little bit more standard by replacing `Error=` by something like `kube-api-error=` or something of the sort. Also... do we want to do this only for `404` errors? This looks like it will persist a file for every resource it couldn't fetch. What do you think?

Reply:
yep, sounds sane. I think forbidden might be interesting for some rules as well?

Reply:
@tjungblu we usually try to fix those in `deploy/role.yaml` here in the operator. The operator should be able to read everything it needs to.

Reply:
right, then let's do 404 only for now?

Reply:
sounds good to me!